codejam-virtual-keyboard/src/js/lib/create-element.js

```diff
@@ -0,0 +1,33 @@
+const is = require('./is');
```

Comment:
It is not clear. I don't understand what the `is` means when I see it first time.
Please do like this:
```javascript
   const { isObject, isPrimitive } = require('./lib/common');
```